Libplanet/Store/Trie/Nodes/INode.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Libplanet.Store.Trie.Nodes
 {
-    internal interface INode
+    public interface INode
```

Comment:
Could you add an XML comment here?

Reply:
I reverted `INode`'s access modifier into `internal` again.